cardano-lib/generic-tracer-config.nix

```diff
@@ -0,0 +1,53 @@
+{
+  # Cardano-tracer will use default values where parameters are set to null.
+  WarnRTViewMissing = null;
```

Comment:
That option doesn't exist.

Reply:
It appears in the code [here](https://github.com/IntersectMBO/cardano-node/blob/master/cardano-tracer/src/Cardano/Tracer/Run.hs#L63-L71), and also appears in the `10.3.1` cardano-tracer config dump upon startup as:
```
cardano-tracer[$PID]: {
  "data":{
    "Config":{
      "WarnRTViewMissing":false,
      ...
    },
  },
  ...
}
```
I also have it in the nixos service option in the node PR.
Happy to remove it, though, if you think it should be gone.

Reply:
Ah, I see, it's outside of the Config object.

Reply:
Removed